ios/Exponent/Versioned/Core/EXVersionManager.m

```diff
@@ -112,10 +112,12 @@ - (void)bridgeWillStartLoading:(id)bridge
      postNotificationName:EX_UNVERSIONED(@"EXReloadActiveAppRequest") object:nil];
   }];
 
+#if DEBUG || RCT_DEV
```

Comment:
should we use this same macro for the header import?

Reply:
No, we don't need to do that because this header is empty when `RCT_DEV` is defined, see https://github.com/facebook/react-native/blob/9b81232ed2cc2beaf06a34ffe2279612f435f931/React/DevSupport/RCTPackagerConnection.h#L12